test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -356,11 +356,11 @@ public void InstallCommand_FromPackagesConfigFileWithExcludeVersion()
                 Directory.CreateDirectory(repositoryPath);
 
                 Util.CreateTestPackage("packageA", "1.1.0", repositoryPath);
-                Util.CreateTestPackage("packageB", "2.2.0", repositoryPath);
+                Util.CreateTestPackage("packageA", "2.2.0", repositoryPath);
```

Comment:
Are these changes intentional? I didn't see them called out in the PR description.

Reply:
[Comment](https://github.com/NuGet/NuGet.Client/pull/5356#discussion_r1293571230) suggested to make package name and project name different. I first changed package name to B, then changed my mind to change project to B.

Reply:
Thanks! I tried to address [this comment](https://github.com/NuGet/NuGet.Client/pull/5356#discussion_r1293571230) but accidently made more changes to other unrelated tests. Fixed.